src/models/User/sendValidationEmail/unit.spec.js

```diff
@@ -4,12 +4,13 @@ import {
   USER_VALIDATION_EMAIL_TEMPLATE_ID
 } from '@constants/emails'
 import sendGrid from '@sendgrid/mail'
+import User from '@models/User'
 import sendValidationEmail, { VALIDATE_PATH } from './index'
 
-const user = {
-  name: 'Foo',
-  email: 'foo@bar.com'
-}
+const user = new User()
```

Comment:
Maybe we could have a constructor? but I guess that works… We’ll get rid of `sendValidationEmail` anyway.